### ISSUE TITLE
finetune will get better performance

### ISSUE THREAD

**Issue by kiinghom** [Coolnesss/fada-pytorch#3]
I found if I use loss = loss_X1  + loss_X2  in the step3 will get better performance.  I don't know why?
